Tools/ML/model.h

```diff
@@ -111,6 +111,39 @@ class OnnxModel
     return evalModel<T>(inputTensors);
   }
 
+  // For multi-dimensional inputs
+  template <typename T>
+  T* evalModel(std::vector<std::vector<T>>& input)
+  {
+    std::vector<Ort::Value> inputTensors;
+
+#if !__has_include(<onnxruntime/core/session/onnxruntime_cxx_api.h>)
+    Ort::MemoryInfo mem_info = Ort::MemoryInfo::CreateCpu(OrtAllocatorType::OrtArenaAllocator, OrtMemType::OrtMemTypeDefault);
+#endif
```

Comment:
You can drop this if guard since it was needed to support an old ONNXRuntime version

Reply:
For now we can also leave it in, not sure how many people might be using an older build stack still